apps/meteor/app/livechat/server/api/v1/contact.js

```diff
@@ -51,19 +51,30 @@ API.v1.addRoute(
 				check(this.queryParams, {
 					email: Match.Maybe(String),
 					phone: Match.Maybe(String),
+					custom: Match.Maybe(String),
 				});
+				const { email, phone, custom } = this.queryParams;
 
-				const { email, phone } = this.queryParams;
-
-				if (!email && !phone) {
+				if (!email && !phone && !custom) {
 					throw new Meteor.Error('error-invalid-params');
 				}
+				const customObj = Object.fromEntries(
+					Array.from(new URLSearchParams(custom)).filter(([k, v]) => typeof k === 'string' && typeof v === 'string'),
+				);
+
+				const foundCF = Object.fromEntries(
+					LivechatCustomField.find(
+						{ scope: 'visitor', searchable: true, _id: RegExp(Object.keys(customObj).join('|')) },
```

Comment:
Instead of regex, use `$in`
```suggestion
						{ scope: 'visitor', searchable: true, _id: { $in: customObj } },
```

Reply:
I'm pretty sure this one is a raw model 👀 so you need to `await` for it